src/components/Comments.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getComments } from "../utils/api";
+
+export const Comments = () => {
+  const [comments, setComments] = useState([]);
+  const { article_id } = useParams();
```

Comment:
I would prefer if we had loading and error state whenever we make a call to an backend